### ISSUE TITLE
Load/Save redesign: local storage won't ask to enter new name

### ISSUE THREAD

**Issue by tempelmann** [ondras/wwwsqldesigner#205]
See this img as I see it in Safari / OS X:

![screen shot 2015-06-16 at 13 28 31](https://cloud.githubusercontent.com/assets/461920/8182079/ed4a19b0-142b-11e5-98f3-dadc9ad492c8.PNG)

This is how it looks initially. As you can see, no name is selected. It would be nice to re-select the last choice.

However, the more critical issue is that there is no way to save to or load from a new name as there is no prompt coming for that. When using the Save or Load button, it simply won't do anything if nothing is selected.

One way to solve this is to add a new item to the popup list, naming it "[New name]", and setting that as the list default (so that there is always something chosen). Still, once a name has been chosen, the next time we open this dialog, it should revert to that last choice.

**Comment by tempelmann** [comment]
Also, I wonder if Quicksave works, too, i.e. will it save to the browser storage when that's chosen last? It certainly should the way the UI looks, but there is no feedback to tell.
(Plus, F2 won't work on everyone's computer, e.g. I've overloaded my F-keys with global shortcuts, but that's not something to worry about - just remember that you may not always have full access to F-keys on a Mac)

**Comment by ondras** [comment]
As far as F2 goes, it works on my machine. (No UI feedback though, which is not nice.)

**Comment by ondras** [comment]
I added the default option so the selectbox shall be never empty.

Persisting the last saved name is tricky - shall this value be shared across different storage backends, or shall it be specific to the Browser storage?

**Comment by tempelmann** [comment]
Well, I wondered the same. That's why I used a separate variable to save the "last saved name" for dropbox and local storage. I don't think it's a problem if using the same name across all storage types, as long as you also remember + restore the last used chosen storage type (because, currently, it always falls back to the server storage mode).

**Comment by ondras** [comment]
Right, the server storage is the default because of the "?keyword=xxx" feature that allows you to permalink to a (server-)saved design.
